### PR TITLE
feat: improve rule security headers on cloudfront

### PR DESCRIPTION
**Pull request by ealain**

### CODE REVIEW

File: packages/core/src/aws-sdk-helpers/cloudFront/fetchDistributionConfig.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
 } from '@aws-sdk/client-cloudfront';
 import { cloudFrontClient } from '../../clients';
 
-const getDistributionConfig = async (
+const fetchDistributionConfig = async (
```
Comment (PChol22) [inline packages/core/src/aws-sdk-helpers/cloudFront/fetchDistributionConfig.ts:7]: I think this change is breaking the rule "cloudFrontNoHttp" due to recent conflicts

**Review by PChol22 — COMMENTED** [review]
A cloudfront rule was merged yesterday and I think it induces small conflicts

+ is the rule still level 1 ? how easy to fix / impactful is it ? what do you think ?

Overall lgtm

File: packages/core/src/aws-sdk-helpers/cloudFront/fetchDistributionConfig.ts (hunk 1 above)
Comment (ealain) [reply packages/core/src/aws-sdk-helpers/cloudFront/fetchDistributionConfig.ts:7]: Fixed

**Comment by ealain** [comment]
> Is the rule still level 1 ? how easy to fix / impactful is it ? what do you think ?

Given `cloudFrontNoHttp` is in level 2, I would go for 3 or 4.

**Review by PChol22 — APPROVED** [review]

**Pull request merged**